pkg/sriov/poolconfig.go

```diff
@@ -255,6 +255,33 @@ func (builder *PoolConfigBuilder) WithMaxUnavailable(maxUnavailable intstrutil.I
 	return builder
 }
 
+// WithRdmaMode method sets rdmaMode to shared/exclusive.
+func (builder *PoolConfigBuilder) WithRdmaMode(mode string) *PoolConfigBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	if mode == "" {
```

Comment:
Do we need this validation? It's already covered by the requirement that it must be either shared or exclusive.

Reply:
Not important. But just to have precise errorMsg in both cases (empty and invalid)